### PR TITLE
[INLONG-784][Doc][Hive] Describe tables using MarkDown

### LINKED ISSUE #784: [Improve][Doc][Hive] Describe tables using MarkDown
### Description

[Improve][Doc][Hive] Describe tables using MarkDown

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by e-mhui**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #784, #776 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/hive.md [hunk 1]
```diff
@@ -61,160 +61,35 @@ CREATE TABLE hiveTableName (
...
-      <td>inlong metric 的标签值，该值的构成为groupId=[groupId]&streamId=[streamId]&nodeId=[nodeId]。</td> 
-     </tr>
-    </tbody>
-</table>
+
+| 参数  | 是否必须 | 默认值 | 数据类型 | 描述  |
+| --- | --- | --- | --- | --- |
+|  connector | 必须  | (none) | String | 指定使用什么连接器，这里应该是 'hive'。 |
+|  default-database | 必须  | (none) | String | 指定数据库名称。 |
+|  hive-conf-dir | 必须  | (none) | String | 本地构建项目可以将hive-site.xml构建到 classpath 中，未来 Dashboard 将支持本地上传能力。 目前通用方式只支持配置已经上传文件后的 HDFS 路径。 |
```
Comment (yunqingmoswu) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/hive.md:69]: Add a space for `hive-site.xml` between chinese words.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/hive.md [hunk 2]
```diff
@@ -61,160 +61,35 @@ CREATE TABLE hiveTableName (
...
+|  sink.partition-commit.trigger | 可选  | (none) | String | 如果表是分区表，可以配置触发模式。如：(process-time) |
+|  partition.time-extractor.timestamp-pattern | 可选  | (none) | String | 如果表是分区表，可以配置时间戳。如：(yyyy-MM-dd) |
+|  sink.partition-commit.delay | 可选  | (none) | String | 如果表是分区表，可以配置延迟时间。如：(10s,20s,1m...) |
+|  sink.partition-commit.policy.kind | 可选  | (none) | String | 分区提交策略通知下游某个分区已经写完毕可以被读取了。 metastore：向 metadata 增加分区。仅 hive 支持 metastore 策略，文件系统通过目录结构管理分区； success-file：在目录中增加 '_success' 文件； 上述两个策略可以同时指定：'metastore,success-file'。 custom：通过指定的类来创建提交策略， 支持同时指定多个提交策略：'metastore,success-file'。 |
+| inlong.metric.labels | 可选  | (none) | String | inlong metric 的标签值，该值的构成为groupId=[groupId]&streamId=[streamId]&nodeId=[nodeId]。 |
```
Comment (yunqingmoswu) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/hive.md:74]: Add a space for `groupId=...` between chinese words.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/hive.md (hunk 1 above)
Comment (e-mhui) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/hive.md]: Done.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/hive.md (hunk 2 above)
Comment (e-mhui) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/hive.md]: Done.

**Review by EMsnap — APPROVED** [review]

**Review by yunqingmoswu — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**